packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -200,20 +257,14 @@ export class SummaryManager extends EventEmitter implements IDisposable {
             return { shouldSummarize: false, stopReason: "parentShouldNotSummarize" };
         } else if (this.disposed) {
             return { shouldSummarize: false, stopReason: "disposed" };
-        } else if (this.orderedClients.getSummarizerCount() > 0) {
```

Comment:
based on the comment i get why we need to remove shouldStart, but is the case handled elsewhere?

Reply:
is it that now refreshSummarizer will detect this state based on this.electedSummarizerManager.electedClientId, so while getShouldSummarizeState will return true, it won't be electect so it it won't run? if so, i wonder if we should rename getShouldSummarizeState to canSummarizeState, or supportsSumarizingState, as whether it should or not is really controlled by election

Reply:
or you could move the newSummarizerClientId logic and event emitting from refreshSummarizer to getShouldSummarizeState

Reply:
We just no longer wait for the summarizer client(s) to leave the quorum.